src/main/java/com/avispl/symphony/dal/avdevices/encoderdecoder/haivision/xencoder/HaivisionXEncoderCommunicator.java

```diff
@@ -228,14 +264,17 @@ public List<Statistics> getMultipleStatistics() {
 		Map<String, String> stats = new HashMap<>();
 		List<AdvancedControllableProperty> advancedControllableProperties = new ArrayList<>();
 
-		roleBased = retrieveUserRole();
-		isConfigManagement = isConfigManagementProperties();
-		populateInformationFromDevice(stats, advancedControllableProperties);
-		if (EncoderConstant.OPERATOR.equals(roleBased) || EncoderConstant.ADMIN.equals(roleBased) && isConfigManagement) {
-			extendedStatistics.setControllableProperties(advancedControllableProperties);
+		if (!isEmergencyDelivery) {
+			roleBased = retrieveUserRole();
+			isConfigManagement = isConfigManagementProperties();
+			localExtendedStatistics = new ExtendedStatistics();
```

Comment:
No need to re-initialize localExtendedStatistics here, because you already assign it to extendedStatistics at line 276.